jenkins/clouds/ibm/weekly/cost_over_usage/Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+pipeline {
```

Comment:
Need also to change jenkins hierarchy to daily, weekly and monthly

Reply:
Already changed